### PR TITLE
[CFP-574] Investigate sessions in CCCD

### PR DESCRIPTION
**Pull request by NenehB**
What
Currently, users can stay signed in to CCCD without their session expiring and being logged out.

Ticket

[CFP-754 - Investigating sessions in CCCD](https://dsdmoj.atlassian.net/browse/CFP-574)

Why
This poses as a security risk as it increases the likelihood that hostile attacks can gain access to accounts that are already logged in to our production environments. 

How
Logging users out after 3 hours of inactivity.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -40,6 +40,7 @@ class User < ApplicationRecord
           :recoverable,
           :rememberable,
           :trackable,
+          :timeoutable,
```
Comment (jrmhaig) [inline app/models/user.rb:43]: Is the timeout limit configured somewhere?

Comment (mpw5) [reply app/models/user.rb:43]: It's configured in the `devise` initializer: [`config/initializers/devise.rb`](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/36155ab6a470e89a3e515619bd53b3e266f3a84d/config/initializers/devise.rb#L153).

**Review by mpw5 — APPROVED** [review]
Great work 🥇

File: app/models/user.rb (hunk 1 above)
Comment (NenehB) [reply app/models/user.rb:43]: yes in the devise.rb file

**Pull request merged**